README.md

```diff
@@ -84,3 +84,32 @@ it's also possible that *transform_all.bat* is looking for a text templating eng
 
 ####Note####
 the installer structure for DynamoCore and DynamoRevit has recently changed - if you have trouble with these instructions please reach out to the Dynamo team for help.
+
+#### Testing latest master with Revit 2020
+* Install Revit 2020 - user Revit Beta Program
+* Build Dynamo master branch
+* Build DynamoRevit master branch. Make sure `dynamo.config` file, which will be in the `bin` folder, is pointing to the `bin` folder of Dynamo that was just built. Your config file should look like below. Note that there's a bug in the system where `dynamo.config` needs to be updated after a successful build
```

Comment:
Can you remove the bug word because I believe dynamo.config is part of the build outcome, it is just like any binary which will be recreated when building.

Reply:
Done. Merging